packages/graph-explorer/src/components/Button/Button.tsx

```diff
@@ -1,63 +1,112 @@
 import { cn } from "@/utils";
-import { useButton } from "@react-aria/button";
-import type { AriaButtonProps } from "@react-types/button";
-import type { ElementType, ForwardedRef, ReactNode, RefObject } from "react";
+import type { ForwardedRef, ReactNode } from "react";
 import { forwardRef } from "react";
-import { useWithTheme } from "@/core";
-import { defaultStyles } from "./Button.styles";
+import { cva, VariantProps } from "cva";
+import { Slot } from "@radix-ui/react-slot";
+
+export const buttonStyles = cva({
```

Comment:
Button styles are somewhat complex since each `variant` should support all the available `color` options. This means all the styles are in the `compoundVariants` section of CVA config.

I tried splitting these styles by `variant`, which made this style code much better, but made reusing the styles in `Link` components  very difficult. So I reverted back to the standard CVA way of doing this.